.zenodo.json

```diff
@@ -0,0 +1,83 @@
+{
+  "description": "DeepReg is a freely available, community-supported open-source toolkit for research and education in medical image registration using deep learning.",
+  "license": "Apache-2.0",
+  "title": "DeepReg: a deep learning toolkit for medical image registration",
+  "upload_type": "software",
+  "keywords": [
+    "Deep Learning",
+    "Image Fusion",
+    "Medical Image Registration",
+    "Neural Networks"
+  ],
+  "creators": [
+    {
+      "orcid": "0000-0002-1184-7421",
+      "name": "Fu, Yunguan"
+    },
+    {
+      "orcid": "0000-0001-5685-971X",
+      "name": "Monta\u00f1a Brown, Nina"
```

Comment:
@NMontanaBrown Is this correct for you?

Reply:
This is my correct name, but can you modify to "Monta\u00f1a-Brown, Nina"? Publishing name, for consistency 😅

Reply:
done ;)